geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -59,17 +60,21 @@
 public class ExecutionHandlerContext extends ChannelInboundHandlerAdapter {
 
   private static final Logger logger = LogService.getLogger();
+  private static final Command TERMINATE_COMMAND = new Command();
 
   private final Client client;
   private final Channel channel;
   private final RegionProvider regionProvider;
   private final PubSub pubsub;
-  private final EventLoopGroup subscriberGroup;
   private final ByteBufAllocator byteBufAllocator;
   private final byte[] authPassword;
   private final Supplier<Boolean> allowUnsupportedSupplier;
   private final Runnable shutdownInvoker;
   private final RedisStats redisStats;
+  private final ExecutorService backgroundExecutor;
+  private final int MAX_QUEUED_COMMANDS = 100;
+  private final LinkedBlockingQueue<Command> commandQueue =
```

Comment:
change this to be unlimited in size

Reply:
I think we wanted to leave this to be bounded. No?

Reply:
Could that cause a deadlock if the queue is full but not moving because the current command is a publish waiting for the latch and the latch is not being cleared by subscribe because it is waiting for a netty worker thread to change to the subscriber thread pool?

Reply:
I've made it configurable via a system property and defaulted it to `1000`.